recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,238 @@
+        "single_header": False,
+        "multithreading": False,
+        "imutable_nodes": False,
+        "web_socket": False,
+        "discovery": True,
+        "discovery_semaphore": True,
+        "discovery_multicast": False,
+        "query": False,
+        "encription": False,
+        "json_support": False,
+        "pub_sub": "None",
+        "data_access": True,
+        "compiled_nodeset_descriptions": True,
+        "namescpae_zero": "FULL",
+        "embedded_profile": False,
+        "typenames": True,
+        "hardening": False,
+        "cpp_compatible": False,
+        "readable_statuscodes": True
+    }
+    generators = "cmake", "cmake_find_package"
+
+    def requirements(self):
+        if self.options.encription:
+            self.requires("mbedtls/2.16.3-gpl")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _download_deps(self, zip_link, dep_name):
+        zip_name = dep_name + ".zip"
+        download(zip_link, zip_name)
+        dep_path = self._source_subfolder + "/deps/" + dep_name
+        unzip(zip_name)
+        full_path = str(urlparse(zip_link).path).split("/")
+        dir_name = full_path[2] + "-" + full_path[4]
+        dir_name = dir_name[:-4]
+        files = os.listdir(dir_name)
+        for f in files:
+            shutil.move(os.path.join(dir_name, f), dep_path)
+        os.unlink(zip_name)
+        shutil.rmtree(dir_name)
+
+    def _download_ua_nodes(self):
+        link = ""
+        if self.version == "1.0.0":
+            link = "https://github.com/OPCFoundation/UA-Nodeset/archive/0777abd1bc407b4dbd79abc515864f8c3ce6812b.zip"
+        elif self.version == "1.0.1":
+            link = "https://github.com/OPCFoundation/UA-Nodeset/archive/0777abd1bc407b4dbd79abc515864f8c3ce6812b.zip"
+        self._download_deps(zip_link=link, dep_name="ua-nodeset")
+
+    def _download_mdnsd(self):
+        link = ""
+        if self.version == "1.0.0":
+            link = "https://github.com/Pro/mdnsd/archive/8fe3a7e7e9d0a9196b126c64f0d1905569b83d40.zip"
+        elif self.version == "1.0.1":
+            link = "https://github.com/Pro/mdnsd/archive/f7f0dd543f12fa7bbf2b667cceb287b9c8184b7d.zip"
+        self._download_deps(zip_link=link, dep_name="mdnsd")
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        if Version(self.version) <= "1.0.0":
+            folder_name = "%s-%s" % (self.name, "1.0")
+        else:
+            folder_name = "%s-%s" % (self.name, self.version)
+        os.rename(folder_name, self._source_subfolder)
+        if Version(self.version) <= "1.0.1":
+            self._download_ua_nodes()
+            self._download_mdnsd()
+        self._patch_sources()
```

Comment:
Please patch in build

Reply:
Resolved in the next commit, just running local tests